subprojects/gradle/cd/src/main/kotlin/com/avito/ci/internal/ReportKey.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.ci.internal
+
+import com.avito.report.model.ReportCoordinates
+import java.util.Locale
+
+/**
+ * RunId doesn't matter here, because it's stable for a single build
+ */
+internal data class ReportKey(val planSlug: String, val jobSlug: String) {
+
+    override fun toString(): String =
+        "${planSlug.capitalize(Locale.getDefault())}${jobSlug.capitalize(Locale.getDefault())}"
```

Comment:
Why do we need implicit capitalization in `toString`?

Reply:
changed to more explicit API